releases/v20.1.0-alpha.20200210.md

```diff
@@ -0,0 +1,351 @@
+- Job can be running but shown in state pending. [#43814][#43814]
+- On Linux machines, we now respect the available memory limit as dictated by the cgroup limits which apply to the cockroach process. [#43137][#43137]
+- Previously, CockroachDB would return incorrect results for some aggregate functions when used as window functions with non-default window frame. This is now fixed. Note that MIN, MAX, SUM, AVG, and "pure" window functions (i.e., non-aggregates) were not affected. [#39308][#39308]
+- Previously, CockroachDB could return an internal error when running a query with a CAST operation (`:::`) in some cases when using the vectorized execution engine. This is now fixed. [#43857][#43857]
+- Previously, a query shutdown mechanism could fail to fully cleanup the infrastructure when the query was executed via the vectorized engine and the query plan contained wrapped row-by-row processors (in v19.2, this applies to Lookup joins and Index joins). This is now fixed. [#43579][#43579]
+- Fixed a bug introduced in v19.2 that would allow foreign keys to use a unique index on the referenced columns that indexed more columns than were included in the columns used in the FK constraint, which allows potentially violating uniqueness in the referenced columns themselves. [#43793][#43793]
+- Ensure that RESTORE cleanup is run exactly once. [#43933][#43933]
+- A benign error previously logged at the ERROR level is now logged at the INFO level behind a verbosity(2) flag. This error might have been observed as "context canceled: readerCtx in Inbox stream handler". [#44020][#44020]
+- A bug causing lost update transaction anomalies was fixed. [#42969][#42969]
+- Previously, an internal error could occur when a query with an aggregate function MIN or MAX was executed via the vectorized engine when the input column was either INT2 or INT4 type. This is now fixed. [#43985][#43985]
+- CDC is no longer susceptible to a bug where a resolved timestamp might be published before all events that precede it have been published in the presence of a Range merge. [#44035][#44035]
+- `cockroach zip` now emits the `goroutine` file in the proper sub-directory when the corresponding call fails with an error. [#44064][#44064]
+- `cockroach debug zip` is again able to operate correctly and continue to iterate over all nodes if one of the nodes does not deliver its goroutine dumps. It would previously prematurely and incorrectly stop with an incomplete dump; this was a regression introduced in v19.2. [#44064][#44064]
+- The file generated by `cockroach gen haproxy` no longer gets an executable bit. The executable bit was previously placed in error. [#44043][#44043]
+- Fixed internal error of the form "x FK cols, only y cols in index" in some cases when inserting into a table with foreign key references. [#44031][#44031]
+- CockroachDB now ensures the import cleanup is only run once. [#43960][#43960]
```

Comment:
AFAICT this doesn't mean anything from the user's perspective.  Could modify to say something like "ensure internal cleanup after `IMPORT` is only run once".

@pbardea does that edit sound ok to you?